go/vt/orchestrator/app/cli.go

```diff
@@ -504,15 +504,7 @@ func Cli(command string, strict bool, instance string, destination string, owner
 		}
 	case registerCliCommand("reattach-replica-primary-host", "Replication, general", `Undo a detach-replica-primary-host operation`):
 		{
-			instanceKey, _ = inst.FigureInstanceKey(instanceKey, thisInstanceKey)
-			if instanceKey == nil {
-				log.Fatal("Cannot deduce instance:", instance)
-			}
-			_, err := inst.ReattachReplicaPrimaryHost(instanceKey)
-			if err != nil {
-				log.Fatal(err)
-			}
-			fmt.Println(instanceKey.DisplayString())
+			fmt.Println("Command deprecated")
```

Comment:
I have removed some API and CLI commands that would have required some changes and that we don't intend to support anyways. I could have removed them as a separate PR, but I elected to do it here as I went along.